src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/AmendReferralController.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller
+
+import mu.KLogging
+import org.springframework.http.HttpStatus
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken
+import org.springframework.web.bind.annotation.PatchMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.AmendComplexityLevelDTO
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.AmendReferralService
+import java.util.UUID
+
+@RestController
+class AmendReferralController(
+  private val amendReferralService: AmendReferralService
+) {
+  companion object : KLogging()
+
+  @PatchMapping("/sent-referral/{referralId}/amend-complexity-level")
+  fun updateComplexityLevel(
+    @PathVariable referralId: UUID,
+    @RequestBody complexityLevel: AmendComplexityLevelDTO,
+    authentication: JwtAuthenticationToken
+  ): AmendComplexityLevelDTO? {
```

Comment:
as we discussed, we can think about returning the changelog from the service. But its ok for now